docs/faq.html

```diff
@@ -51,6 +51,39 @@ <h2 class="title-2">
   </p>
 </section>
 
+<section class="mt4">
+  <h2 class="title-2"><a name="verify">How can I verify the download of Please?</a></h2>
```

Comment:
nit: `<h2 id="verify" class="title-2">How can I verify the download of Please?</h2>` will make it consistent with similar usages in other places.

Reply:
Ah thanks, I never knew that worked that way!